13/umbraco-deploy/getting-started/deploy-settings.md

```diff
@@ -372,6 +380,29 @@ In production this setting shouldn't be changed from it's default value of `fals
 
 If attempting a one-off, large transfer operation, before a site is live, you could set this value to `true`. That would omit the firing and handling of these notifications and remove their performance overhead. Following which you would need to ensure to rebuild the cache and search index manually via the backoffice _Settings_ dashboards.
 
+### Suspensions
+
+Deploy operations suspend scheduled publishing, Examine indexing, document cache and/or signature database update events by default. You can amend this behavior for all supported or specific operations using these settings.
+
+Each setting within this section represents a Deploy operation. For each, the suspensions that are carried out can be amended with one or more of following values:
+
+-  `DiskRead` - `None, ScheduledPublishing, Examine, DocumentCache, All`,
+-  `PartialRestore` - `None, ScheduledPublishing, Examine, DocumentCache, All`,
+-  `Restore` - `None, ScheduledPublishing, Examine, DocumentCache, Signatures|All`,
+-  `Deploy` - `None, ScheduledPublishing, All`,
+-  `Import` - `None, ScheduledPublishing, Examine, DocumentCache, All`,
+-  `Export` - `None, ScheduledPublishing, All`
+
+The default value for all suspension settings is `All`.
```

Comment:
This should also mention you can set the values for all operations by setting the `Suspensions` value to a valid value (instead of an object), e.g.:
```json
  "Suspensions": "ScheduledPublishing, DocumentCache, Signatures"
```

Reply:
Have added this - thanks.